react/features/toolbar/functions.web.js

```diff
@@ -0,0 +1,159 @@
+/* @flow */
+
+import SideContainerToggler
+    from '../../../modules/UI/side_pannels/SideContainerToggler';
+
+import defaultToolbarButtons from './defaultToolbarButtons';
+
+declare var AJS: Object;
+declare var interfaceConfig: Object;
+declare var $: Function;
```

Comment:
we keep declaring these (and similar globals) on many files. Can you look into putting all those into a file which flow uses automagically? Similarly to how flow-typed does it for other projects.